extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -733,33 +749,35 @@ export class AmpAnalytics extends AMP.BaseElement {
  */
 class AmpAnalyticsFrames {
   constructor() {
-    this.requestsFrameMap = new Map();
+    this.requestsFrameMap = map();
   }
 
   /**
    * Create a cross-domain iframe
-   * @param {!Element} parent  The document node of the parent page
+   * @param {!HTMLDocument} parent  The document node of the parent page
    * @param {!string} requestsFrameUrl  The URL, including data hash if
    * applicable, of the cross-domain iframe
    * @return {!Element}
    */
   create(parent, requestsFrameUrl) {
-    let frame = createElementWithAttributes(parent, 'iframe', {
+    // Warning: the scriptSrc URL below is only temporary. Don't check
+    // in before getting resolution on that.
+    const frame = createElementWithAttributes(parent, 'iframe', {
       sandbox: 'allow-scripts',
       name: JSON.stringify({
-        'scriptSrc': '/examples/analytics-3p-remote-frame-helper.js'
+        'scriptSrc': '/examples/analytics-3p-remote-frame-helper.js',
       }),
     });
-    frame.onload = () => {
+    loadPromise(frame).then(() => {
```

Comment:
Linter seems wary of this. Is this how you intended for loadPromise to be used here, @jridgewell ?

`[16:45:49] ==========
[16:45:49] Found forbidden: "{loadPromise" in extensions/amp-analytics/0.1/amp-analytics.js:19:8
[16:45:49] Most users should use BaseElement…loadPromise.
[16:45:49] ==========
[16:45:49] Found forbidden: " loadPromise" in extensions/amp-analytics/0.1/amp-analytics.js:771:4
[16:45:49] Most users should use BaseElement…loadPromise.
[16:45:49] ==========
[16:45:53] Please remove these usages or consult with the AMP team.`